cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorCustomCertTest.java

```diff
@@ -196,7 +204,7 @@ public Secret getTlsSecret() {
     }
 
     public String getTlsThumbprint()    {
-        return "vjPd/D/f0/X3yqitf65yoUZbyeWnQU4cPDJGbr7GA7I=";
+        return "{external=COWn2zWLZMhoewfrmSTfUeKlQPifBKekyXzjm2iGTuc=, tls=vjPd/D/f0/X3yqitf65yoUZbyeWnQU4cPDJGbr7GA7I=}";
```

Comment:
What changed here?

Reply:
We used to have one thumbprint annotation for TLS listener and one for Externaland that was fine since we had only these two listeners where it was needed. But now we have X possible listeners with their own thumbprints. So we use one annotation for all of them.